### PR TITLE
Feat testing skeleton

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Testing practices and guidelines
+
+TODO: Follow https://google.github.io/googletest/
```
Comment (MicheleMichetti) [inline test/README.md]: I would delete "TODO" before merging

Comment (ghost) [reply test/README.md]: It is a good point

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+services:
+  bash:
```
Comment (MicheleMichetti) [inline docker-compose.yml]: bash -> build_container (check syntax)

File: tests/CMakeLists.txt
Comment (MicheleMichetti) [inline tests/CMakeLists.txt:1]: Link the libraries of the game, otherwise the tests won't find them.

**Review by MicheleMichetti — CHANGES_REQUESTED** [review]
See single comments

File: docker-compose.yml (hunk 2 above)
Comment (ghost) [reply docker-compose.yml]: Fixed

File: tests/CMakeLists.txt
Comment (ghost) [reply tests/CMakeLists.txt:1]: Fixed

**Review by MicheleMichetti — APPROVED** [review]

**Pull request merged**